### PR TITLE
[SOAR-733] - Azure ad admin revoke session

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Description: 

This PR adds the Revoke Sign-In Sessions action to Azure AD Admin

## Validation: 

```
icon-rmt-mbp-5357:azure_ad_admin jmcadams$ icon-validate .
[*] Validating plugin at .

[*] Running Integration Validators...
[*] Executing validator HelpValidator
[*] Executing validator ChangelogValidator
[*] Executing validator RequiredKeysValidator
[*] Executing validator UseCaseValidator
[*] Executing validator SpecPropertiesValidator
[*] Executing validator SpecVersionValidator
[*] Executing validator FilesValidator
[*] Executing validator TagValidator
[*] Executing validator DescriptionValidator
[*] Executing validator TitleValidator
[*] Executing validator VendorValidator
[*] Executing validator DefaultValueValidator
[*] Executing validator IconValidator
[*] Executing validator RequiredValidator
[*] Executing validator VersionValidator
[*] Executing validator DockerfileParentValidator
[*] Executing validator LoggingValidator
[*] Executing validator ProfanityValidator
[*] Executing validator AcronymValidator
[*] Executing validator JSONValidator
[*] Executing validator OutputValidator
[*] Executing validator RegenerationValidator
[*] Executing validator HelpInputOutputValidator
[*] Executing validator SupportValidator
[*] Executing validator RuntimeValidator
[*] Plugin successfully validated!

----
[*] Total time elapsed: 324.671ms
rmt-mbp-5357:azure_ad_admin jmcadams$
```

## Testing: 
```
rmt-mbp-5357:azure_ad_admin jmcadams$ ./run.sh -R tests/revoke_sign_in_sessions.json -j
Running: cat tests/revoke_sign_in_sessions.json | docker run --rm   -i rapid7/azure_ad_admin:2.1.1  run | grep -- ^\{ | jq -r '.body | try(.log | split("\n") | .[]),.output'
Connect: Connecting...
Updating auth token...
Getting token from: https://login.microsoftonline.com/5c824599-dc8c-4d31-96fb-3b886d4f8f10/oauth2/token
Starting new HTTPS connection (1): login.microsoftonline.com:443
https://login.microsoftonline.com:443 "POST /5c824599-dc8c-4d31-96fb-3b886d4f8f10/oauth2/token HTTP/1.1" 200 1704
Authentication request status: 200
Authentication Token: ****************WRqSQ
rapid7/Azure AD Admin:2.1.1. Step name: revoke_sign_in_sessions
Revoking sign in sessions for user: jmcadams@komanddev.onmicrosoft.com
Starting new HTTPS connection (1): graph.microsoft.com:443
https://graph.microsoft.com:443 "POST /v1.0/5c824599-dc8c-4d31-96fb-3b886d4f8f10/users/jmcadams@komanddev.onmicrosoft.com/revokeSignInSessions HTTP/1.1" 200 194
Connect: Connecting...
Updating auth token...
Getting token from: https://login.microsoftonline.com/5c824599-dc8c-4d31-96fb-3b886d4f8f10/oauth2/token
Authentication request status: 200
Authentication Token: ****************WRqSQ
rapid7/Azure AD Admin:2.1.1. Step name: revoke_sign_in_sessions
Revoking sign in sessions for user: jmcadams@komanddev.onmicrosoft.com

{
  "success": true
}
```

### CODE REVIEW

**Comment by jmcadams-r7** [comment]
Revoked app secret in Azure

File: azure_ad_admin/help.md [hunk 1]
```diff
@@ -31,10 +31,55 @@ The connection configuration accepts the following parameters:
...
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|user_id|string|None|True|User ID|None|user@example.com|
+
+Example input:
+
+```
+{
+  "user_id": "user@example.com
```
Comment (tylerslijboom-r7) [inline azure_ad_admin/help.md]:
```suggestion
  "user_id": "user@example.com"
```

File: azure_ad_admin/plugin.spec.yaml [hunk 2]
```diff
@@ -545,6 +545,23 @@ actions:
         description: Was operation successful
         type: boolean
         required: true
+  revoke_sign_in_sessions:
+    title: Revoke Sign-In Sessions
+    description: Invalidates all the refresh tokens issued to applications for a user (as well as session cookies in a user's browser), by resetting the signInSessionsValidFromDateTime user property to the current date-time
```
Comment (tylerslijboom-r7) [inline azure_ad_admin/plugin.spec.yaml:550]: This seems a bit too technical, especially compared to the other action descriptions.  
```suggestion
    description: Logs the user out of all AD Azure applications.  The user must re-login to all applications.
```

File: azure_ad_admin/tests/revoke_sign_in_sessions.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "body": {
+        "action": "revoke_sign_in_sessions",
+        "input": {
+            "user_id": "jmcadams@komanddev.onmicrosoft.com"
```
Comment (tylerslijboom-r7) [inline azure_ad_admin/tests/revoke_sign_in_sessions.json]:
```suggestion
            "user_id": "user@example.com"
```

File: azure_ad_admin/help.md (hunk 1 above)
Comment (jmcadams-r7) [reply azure_ad_admin/help.md]: Fixed.

**Pull request merged**